shell/platform/android/test/io/flutter/plugin/platform/PlatformPluginTest.java

```diff
@@ -551,11 +552,34 @@ public void setRequestedOrientationFlutterFragment() {
     PlatformPlugin platformPlugin =
         new PlatformPlugin(mockFragmentActivity, mockPlatformChannel, mockPlatformPluginDelegate);
 
-    platformPlugin.mPlatformMessageHandler.setPreferredOrientations(0);
+    platformPlugin.mPlatformMessageHandler.setPreferredOrientations(new AndroidOrientation(0, 0));
 
     verify(mockFragmentActivity, times(1)).setRequestedOrientation(0);
   }
 
+  @Test
+  public void getRequestedOrientationFlutterFragment() {
+    FragmentActivity mockFragmentActivity = mock(FragmentActivity.class);
+    PlatformChannel mockPlatformChannel = mock(PlatformChannel.class);
+    PlatformPluginDelegate mockPlatformPluginDelegate = mock(PlatformPluginDelegate.class);
+    when(mockPlatformPluginDelegate.popSystemNavigator()).thenReturn(false);
+    PlatformPlugin platformPlugin =
+        new PlatformPlugin(mockFragmentActivity, mockPlatformChannel, mockPlatformPluginDelegate);
+
+    for (int requestedAndroidOrientation = 0;
+        requestedAndroidOrientation <= 15;
```

Comment:
This formatting looks odd can this for loop fit on one line if you make the variable name shorter? 
Can you please add a comment describing where 0 and 15 come from? I think it is 1111 in bits but if you are referring to binary values it might be better to us hex.

Reply:
I introduced the `mostSignificantBit` function which (I hope) helps with the readability here and also removes a "magic" number